command/daemon/command.go

```diff
@@ -142,6 +144,7 @@ func (c *command) Execute(cmd *cobra.Command, args []string) {
 			newServer.Shutdown()
 		}()
 
+		wg.Wait()
 		os.Exit(0)
 	}()
 
```

Comment:
I think in the line 150 below we shouldn't exit with `0`. 
https://github.com/giantswarm/microkit/pull/119/files#diff-00e7385156c4149158201c8125cc14d5L150

When we reach there this means SIGINT/SIGTERM was send second time and we don't shutdown cleanly.